src/Temporalio/Workflows/WorkflowDefinitionOptions.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using Temporalio.Common;
+
+namespace Temporalio.Workflows
+{
+    /// <summary>
+    /// Additional options for a workflow definition that may be returned by
+    /// a function annotated with <see cref="WorkflowDynamicOptionsAttribute"/>.
+    /// </summary>
+    public class WorkflowDefinitionOptions
+    {
+        /// <summary>
+        /// Gets the types of exceptions that, if a workflow-thrown exception extends, will
+        /// cause the workflow/update to fail instead of suspending the workflow via task failure.
+        /// These are applied in addition to
```

Comment:
Somewhere need to clarify if set, this replaces any value set on the `WorkflowAttribute`